activerecord/lib/active_record/encryption/encryptor.rb

```diff
@@ -78,6 +86,11 @@ def binary?
         serializer.binary?
       end
 
+      # Returns whether the text is compressed or not
```

Comment:
Was this method moved above private just for use in Rails? If an app is unlikely to call this, it should be `nodoc` instead of documented.